pkgs/by-name/xr/xremap/package.nix

```
@@ -0,0 +1,88 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+}:
+let
+  pname = "xremap";
+  version = "0.10.10";
+
+  src = fetchFromGitHub {
+    owner = "xremap";
+    repo = pname;
+    tag = "v${version}";
+    hash = "sha256-U2TSx0O2T53lUiJNpSCUyvkG4Awa0+a4N6meFn09LbI=";
+  };
+
+  cargoHash = "sha256-8IVexszltzlBBShGkjZwyDKs02udrVpZEOwfzRzAraU=";
+
+  buildXremap =
+    {
+      suffix ? "",
+      features ? [ ],
+      descriptionSuffix ? "",
+    }:
+    rustPlatform.buildRustPackage {
```

```suggestion
    }:
    assert descriptionSuffix != "" && features != [ ];
    rustPlatform.buildRustPackage {
```
If I understand correctly the package wouldn't work with no features enabled